charts/images.yaml

```diff
@@ -235,7 +235,7 @@ images:
 - name: fluent-bit-plugin-installer
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
-  tag: "v0.41.0"
+  tag: "v0.42.0"
```

Comment:
From https://github.com/gardener/gardener/pull/6458, I think that the loki-curator image has to be bumped to `v0.42.0` as well.